server/app/services/program/StatusDefinitions.java

```diff
@@ -0,0 +1,71 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import services.LocalizedStrings;
+
+public class StatusDefinitions {
+
+  @JsonProperty("statuses")
+  private ImmutableList<Status> statuses;
+
+  @JsonCreator
+  public StatusDefinitions(@JsonProperty("statuses") ImmutableList<Status> statuses) {
+    this.statuses = ImmutableList.copyOf(statuses);
```

Comment:
suggestion (blocking): since the argument is immutable there's no need to copy it

Reply:
done